backend/run.sh

```diff
@@ -19,9 +19,21 @@ fi;
 
 # Migrate first
 python manage.py migrate
+
 python manage.py drop_deprecated_api_schema_and_views && 
-  python manage.py drop_api_schema && 
-  python manage.py create_api_schema && 
-  python manage.py create_api_views
+python manage.py drop_api_views &&
+python manage.py drop_api_schema && 
+python manage.py create_api_schema && 
+python manage.py create_api_views
+
+if [[ ("${ENV}" == "LOCAL" || "${ENV}" == "STAGING") && "$LOAD_TEST_DATA" == 1 ]];
```

Comment:
This block looks like it was maybe used for experimenting?

Reply:
This block is necessary to control data loading; we run `run.sh` in our local container stack. `docker-compose-data.yml` sets the LOAD_TEST_DATA env var. I think the other ENV tests are not needed, but this controls whether or not `run.sh` is running under our "normal" stack, or a stack that loads the historic data.

Reply:
I should've highlighted the whole thing, but I was referring to the hardcoded dbkey list on line 32. Is that dbkey list stable / did you have a particular characteristic in mind when picking them or are they just convenient examples? Maybe just a comment would suffice for future travelers.